_includes/v21.2/sidebar-data/deploy.json

```diff
@@ -0,0 +1,416 @@
+{
+  "title": "Deploy",
+  "is_top_level": true,
+  "items": [
+    {
+      "title": "Choose a Deployment Strategy",
+      "items": [
+        {
+          "title": "CockroachDB Cloud",
+          "items": [
+            {
+          	  "title": "Create an Account",
+          	  "urls": [
+          	    "/cockroachcloud/create-an-account.html"
+          	  ]
+          	},
+            {
+              "title": "CockroachDB Serverless",
+              "items": [
+                {
+              	  "title": "Create a CockroachDB Serverless Cluster",
+              	  "urls": [
+              	    "/cockroachcloud/create-a-serverless-cluster.html"
+              	  ]
+              	},
+                {
+              	  "title": "Connect to Your Cluster",
+              	  "urls": [
+              	    "/cockroachcloud/connect-to-a-serverless-cluster.html"
+              	  ]
+              	}
+              ]
+            },
+            {
+              "title": "CockroachDB Dedicated",
+              "items": [
+                {
+              	  "title": "Create a CockroachDB Dedicated Cluster",
+              	  "urls": [
+              	    "/cockroachcloud/create-a-serverless-cluster.html"
+              	  ]
+              	},
+                {
+              	  "title": "Connect to Your Cluster",
+              	  "urls": [
+                    "/cockroachcloud/connect-to-your-cluster.html"
+              	  ]
+              	},
+                {
+              	  "title": "Move into Production",
+              	  "urls": [
+              	    "/cockroachcloud/production-checklist.html"
+              	  ]
+              	}
+              ]
+            },
+            {
+              "title": "CockroachDB Cloud FAQs",
+              "urls": [
+                "/cockroachcloud/serverless-faqs.html",
+                "/cockroachcloud/frequently-asked-questions.html"
+              ]
+            }
+          ]
+        },
+        {
+          "title": "CockroachDB Self-Hosted",
+          "urls": [
+            "/${VERSION}/index.html"
+          ],
+          "items": [
+            {
+              "title": "Get Started",
+              "items": [
+                {
+                  "title": "Install CockroachDB",
+                  "urls": [
+                    "/${VERSION}/install-cockroachdb.html",
+                    "/${VERSION}/install-cockroachdb-mac.html",
+                    "/${VERSION}/install-cockroachdb-linux.html",
+                    "/${VERSION}/install-cockroachdb-windows.html"
+                  ]
+                },
+                {
+                  "title": "Start a Local Cluster",
+                  "items": [
+                    {
+                      "title": "From Binary",
```

Comment:
I wonder if these titles should include "Start a Local Cluster". I worry the SEO isn't going to be work well because important context is missing.

Reply:
I don't know how important navigation link titles are compared to the title in the front-matter of the linked file for SEO purposes. My impression is that the navigation isn't nearly as important, but I could well be mistaken.

Reply:
For the specific page, I don't think there are many, if any, SEO considerations to the sidenav. From a search engine perspective, it's just more links on the page. That said, as a team, we do need to align on our conventions around filename/page title/sidenav link. Lauren created a docs issue for us.